src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_redirect.html.twig

```diff
@@ -39,7 +39,7 @@
             <div class="block-exception">
                 <h1>Redirection Intercepted</h1>
 
-                {% set absolute_url = host in location ? location : host ~ location %}
+                {% set absolute_url = host in location or not (location starts with '/') ? location : host ~ location %}
```

Comment:
```suggestion
                {% set absolute_url = absolute_url(location) %}
```

Seems to do perfectly the job :) 

Passing UrlHelper into the controller would require a lot of dependency injection, i'm not sure it's worth it as it only would be used to format a link...

Reply:
Indeed, thanks.